.env

```diff
@@ -1,3 +1,3 @@
-VUE_APP_HUB_URL=https://testnet.snapshot.org
-VUE_APP_IPFS_GATEWAY=gateway.ipfs.io
-VUE_APP_DEFAULT_NETWORK=1
+VITE_APP_HUB_URL=https://hub.snapshot.org
```

Comment:
Any reason to change this? This is the default network if we merge with that it show prod instead of testnet on demo.snapshot.org.